client/components/PubEdge/pubEdge.scss

```diff
@@ -0,0 +1,78 @@
+@import 'styles/variables.scss';
```

Comment:
We could automate this by adding

```json
	"overrides": [
		{
			"files": "*.scss",
			"options": {
				"useTabs": false
			}
		}
	]
```

to the prettier config file. Thoughts?

Reply:
Ha - I didn't realize that preference/assumption (stating that just in case I unintentionally set that precedent). I've just been doing whatever prettier does by default - though fwiw, my editor (and brain) defaults to preferring tabs. No strongly held opinions on the matter other than consistency through the codebase and enforcement through eslint and prettier configs.